lib/ansible/modules/utilities/logic/wait_for.py

```diff
@@ -448,13 +442,13 @@ def main():
         module.fail_json(msg="state=stopped should only be used for checking a port in the wait_for module")
     if path and state == 'drained':
         module.fail_json(msg="state=drained should only be used for checking a port in the wait_for module")
-    if params['exclude_hosts'] is not None and state != 'drained':
+    if module.params['exclude_hosts'] is not None and state != 'drained':
         module.fail_json(msg="exclude_hosts should only be with state=drained")
-    for _connection_state in params['active_connection_states']:
+    for _connection_state in module.params['active_connection_states']:
         try:
             get_connection_state_id(_connection_state)
         except:
-            module.fail_json(msg="unknown active_connection_state ("+_connection_state+") defined")
+            module.fail_json(msg="unknown active_connection_state (%s) defined" % _connection_state)
 
     msg = params['msg']
```

Comment:
IMHO this should be
~~~ diff
-    msg = params['msg']
+    msg = module.params['msg']
~~~